code/game/objects/items/melee/misc.dm

```diff
@@ -174,13 +174,14 @@
 	var/cooldown_check = 0 // Used interally, you don't want to modify
 
 	var/cooldown = 40 // Default wait time until can stun again.
-	var/stun_time_carbon = 60 // How long we stun for - 6 seconds.
+	var/stun_time_carbon = 20 // How long we stun for - 2 seconds.
 	var/stun_time_silicon = 0.60 // Multiplier for stunning silicons; if enabled, is 60% of human stun time.
+	var/stamina_damage = 65 // Do we deal stamina damage - two hits stamina crits.
```

Comment:
The telescopic baton inherits this, you should really consider setting the stamina damage to 0 for the telescopic baton unless you intend to buff it by a lot. Also its stamina damage will apply even when its folded in currently so you definitely need to fix that.

Reply:
I lied, I tested it and it doesn't apply stamina damage when retracted, although it is still a large buff with the stamina damage added, so I want to verify you're aware of what you're doing.

Reply:
The effects are intentional, numbers can be tweaked as needed based on feedback.